scrapy 2.6.2

skupr-anaconda opened the pull request:
License: https://github.com/scrapy/scrapy/blob/2.6.2/LICENSE
Changelog: https://docs.scrapy.org/en/2.6/news.html#scrapy-2-6-2-2022-07-25
Requirements: https://github.com/scrapy/scrapy/blob/2.6.2/setup.py

Actions: 
1. Fix `libxml2` pinning and add a comment about exclusions, see:
- https://github.com/scrapy/scrapy/pull/5208 (created on 16 Jul 2021)
- https://github.com/AnacondaRecipes/lxml-feedstock/commit/173869446fd13f3189291ba14af27958cd1bc1c6
- libxml2 exclusions due to https://bugs.launchpad.net/lxml/+bug/1928795

skupr-anaconda committed:
Fix libxml2 pinning and add a comment about expulsions

varlackc reviewed (commented):
We will need to add a `license_family` to the recipe, other than that everything else looks good.

skupr-anaconda committed:
Fix license, add license_family

skupr-anaconda commented:
@varlackc I've added `license_family` and fixed license **name** as `BSD-3-Clause` https://github.com/scrapy/scrapy/blob/2.6.2/LICENSE (checked it here https://spdx.org/licenses/)

pseudoyim reviewed (approved):
👍 LGTM
